plugins/tours/tests/phpunit/unit/_renderCustomColumnContent.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for _render_custom_column_content().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Mockery as m;
+use Brain\Monkey;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\_render_custom_column_content;
+
+/**
+ * Class Tests_RenderCustomColumnContent
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Unit
+ * @group   tours
+ * @group   admin
+ */
+class Tests_RenderCustomColumnContent extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var Mockery
+	 */
+	protected $post;
+
+	/**
+	 * The post ID
+	 *
+	 * @var integer
+	 */
+	protected $ID;
+
+	/**
+	 * The post type
+	 *
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/admin/wp-list-table.php';
+
+		$this->post = m::mock( 'post' );
+		$this->post->ID = (int) 99;
+		$this->post->post_type = 'tours';
```

Comment:
'post' is not a valid class. Rather, it's `WP_Post`. However, in a unit test, this class does not get loaded into memory. 

What can you do.? If you need a post object in a unit test, you can create it like this:

```php
$this->post = (object) [
	'ID' => 99,
	'post_type' => 'tours',
	'post_name' => 'tour-123'
	'post_title' => 'Tour 123',
];
```

However, in this test, you don't need the post object. Instead, you can deal with the pieces of data you need within each test.